R/CohortComparisonDiagnostics.R

```diff
@@ -67,18 +67,15 @@ computeCohortOverlap <- function(connectionDetails = NULL,
     return(tidyr::tibble())
   }
   
-  sql <- SqlRender::loadRenderTranslateSql(
-    "CohortOverlap.sql",
-    packageName = utils::packageName(),
-    dbms = connection@dbms,
+  sql <- SqlRender::readSql(system.file("sql/sql_server/CohortOverlap.sql", 
```

Comment:
instead of slashes use separate strings e.g.

```
system.file("sql", "sql_server", CohortOverlap.sql", package = utils::packageName())
```

Reply:
Use of sql_server folder directly is not advised, the way `loadRenderTranslateSql` worked before is correct.